src/domain/constants.js

```diff
@@ -122,6 +122,11 @@ export const IMAGE_SOURCE_TYPE = {
   external: 'external'
 };
 
+export const MIDI_SOURCE_TYPE = {
```

Comment:
You could move this into `src/plugins/midi-piano` in a `constants.js` file, as it is only used by this plugin. We do the same for the other plugins which have their own non-shared constants.

Reply:
Sorry, actually I see that this should actually be used outside of the plugin folder.
See my comment here: https://github.com/educandu/educandu/pull/649/files#r989035573